### DISCUSSION TITLE
The best ever voice generation model in existence just got released as opensource and free to use - how long until we get an extension for webui?

### DISCUSSION THREAD

**Discussion by sarutobiumon** [oobabooga/text-generation-webui#5164] [General]
Truly incredible. Someone just released pretty much ElevenLabs but for free as a local model:

https://github.com/myshell-ai/OpenVoice

https://www.youtube.com/watch?v=HU0PWLbiSWI

**Comment by AndiLoU** [comment]
Holy crap, this sounds awesome!

**Comment by TheLounger** [comment]
Doesn't look all that positive beside their own demos.
The quality of his cloned voice [in the video](https://youtu.be/HU0PWLbiSWI?feature=shared&t=946) you posted is questionable.
Many saying it's nothing like the demo: https://github.com/myshell-ai/OpenVoice/issues/43
It seems to be using 4GB+ RAM as well: https://github.com/myshell-ai/OpenVoice/issues/48

**Reply by laserbeak** [reply]
Still interested.
